GitUpKit/Components/GIDiffFilesViewController.m

```diff
@@ -242,6 +196,10 @@ - (void)setSelectedDeltas:(NSArray*)deltas {
   [_tableView scrollRowToVisible:indexes.firstIndex];
 }
 
+- (GIDiffFileData*)objectAtIndexedSubscript:(NSInteger)index {
```

Comment:
Eh, this is really weird to have on a non indexable data-type class. Can we just add generics to the mutable array instead?

Reply:
> so this can be accessed outside this class

It's not even outside the file, it's exclusively from one `GIDiffFilesViewController` to another, it was just kvetching about accessing the ivar.

I'll revisit it, thanks!

Reply:
Done!